Swift/Tests/TLSIdentityTest.swift

```
@@ -149,7 +149,7 @@ class TLSIdentityTest: CBLTestCase {
     }
     
     func testCreateGetDeleteServerIdentity() throws {
-        if (!keyChainAccessAllowed) { return }
+        try XCTSkipUnless(keyChainAccessAllowed, "Skipping, keychain is not allowed in this config")
```

Don't we need to ignore exception here?

---

Exception pointer is only hit on ObjC, and not in Swift.

---

I don't know. Can you wrap this in a function such as checkKeyChainAccess()?